commit ab7627ff03013f126826d31bf031759fb395d523
Author: Matthew Farrellee <mfarrellee@redhat.com>
Date:   Wed Aug 6 11:09:23 2008 -0500

    Detect GCC 4.3.1, seems that the default should be to parse gcc_vers

diff --git a/src/configure.ac b/src/configure.ac
index 566238e..db0eea6 100644
--- a/src/configure.ac
+++ b/src/configure.ac
@@ -884,6 +884,12 @@ case "$gcc_vers" in
     CompilerMinor="2"
     CompilerPatch="1"
     ;;
+ "4.3.1" )
+    CompilerKind="GCC"
+    CompilerMajor="4"
+    CompilerMinor="3"
+    CompilerPatch="1"
+    ;;
  * )
     if test "x$enable_gcc_version_check" = "xyes"; then
       AC_MSG_RESULT([ERROR])
